src/wp-includes/class-wp-customize-manager.php

```diff
@@ -5673,7 +5673,7 @@ public function register_controls() {
 		$section_description .= __( 'Add your own CSS code here to customize the appearance and layout of your site.' );
 		$section_description .= sprintf(
 			' <a href="%1$s" class="external-link" target="_blank">%2$s<span class="screen-reader-text"> %3$s</span></a>',
```

Comment:
Ideally I'd add a translator comment for the 3 placeholder :)

Reply:
The line with the placeholders here is not translatable, so it does not need a translator comment. The first commit in this PR actually looks good to me as is :)